servers/register_server_types.cpp

```diff
@@ -92,8 +83,22 @@
 #include "xr/xr_positional_tracker.h"
 #include "xr_server.h"
 
+#include "navigation_server_2d.h"
+#include "physics_2d/godot_physics_server_2d.h"
+#include "physics_server_2d.h"
+#include "physics_server_2d_wrap_mt.h"
+#include "servers/extensions/physics_server_2d_extension.h"
+
+#ifndef _3D_DISABLED
```

Comment:
I think we should make explicit with a comment why those two blocks are separate from the rest.
```suggestion
// 3D physics.
#ifndef _3D_DISABLED
```